### ISSUE TITLE
I almost got 1.19.3 to work

### ISSUE THREAD

**Issue by OmegaXVVII** [Trainguy9512/locomotion#101]
Ok first of all how did Asian Half Squat get it to work for 1.19.3? And also I almost got it to work but ever time I do it I just have l loom_mappings_1_19_3_layered_hash_2198_v2.com.terraformersmc-modmenu-5.0.2.jars

**Comment by OmegaXVVII** [comment]
Also what is this? minecraft-project-@-merged-named.jar

**Comment by OmegaXVVII** [comment]
Ok I got it to work

**Comment by AndrexSaberz98** [comment]
how

**Comment by OmegaXVVII** [comment]
But the only problem is the running and walking animation are playing in a fixed rotation and on top of that it wont stop running even when I stop moving

**Comment by OmegaXVVII** [comment]
It kind of works

**Comment by OmegaXVVII** [comment]
Lemme use a different version of the mod rq

**Comment by AndrexSaberz98** [comment]
yeah same for me like I'm always looking forward no matter what way I am actually looking and I'm constantly in the running animation

**Comment by OmegaXVVII** [comment]
Wait lemme move 3d layers out

**Comment by OmegaXVVII** [comment]
Imma see if it works then

**Comment by OmegaXVVII** [comment]
Still not working

**Comment by OmegaXVVII** [comment]
Imma use a different version

**Comment by OmegaXVVII** [comment]
None of them work unfortunately
I think those were just testing for if it would actually function correctly and the animations would run instead of being a actual working download

**Comment by Woodgoldtwo** [comment]
The mod is still in pre-alpha, most of the animations do not work correctly yet

**Comment by Woodgoldtwo** [comment]
The mod in the video was a stolen and ported version of pre-rewrite Animation Overhaul

**Comment by OmegaXVVII** [comment]
Aw dang, why cant we have the ported version? Cant the guy ask for permission?
